pkg/config/config.go

```diff
@@ -118,3 +118,14 @@ func ParseConfig() error {
 	logrus.Debugf("params = %v ", Parameters)
 	return nil
 }
+
+// Copy will create a flat copy of GenericMap
+func (m GenericMap) Copy() map[string]interface{} {
```

Comment:
Should the input and output be both `GenericMap` ??? 

i.e. s/ `map[string]interface{}` / `GenericMap`

Reply:
sounds good :+1: